docs/configuration/settings.rst

```diff
@@ -0,0 +1,586 @@
+
+There is a way to enable this behaviour (it is deactivated by default for
+obvious security reasons). In the `.ini` file:
+
+.. code-block :: ini
+
+    kinto.flush_endpoint_enabled = true
+
+Then, issue a `POST` request to the `/__flush__` endpoint to flush all
+the data.
+
+Project information
+===================
+
++---------------------------------------+--------------------------------------------------------------------------+
+| Setting name                          | What does it do?                                                         |
++=======================================+==========================================================================+
+| cliquet.error_info_link               | The HTTP link returned when uncaught errors are triggered on the server. |
+| ``https://github.com/kinto/kinto/     |                                                                          |
+| issues/``                             |                                                                          |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.project_docs                  | The URL where the documentation of the Kinto instance can be found. Will |
+| ``'http://kinto.readthedocs.org'``    | be returned in :ref:`the hello view <api-utilities>`.                    |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.project_name ``'kinto'``      | The project name returned in :ref:`the hello view <api- utilities>`.     |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.project_version ``''``        | The version of the project. Will be returned in :ref:`the hello view     |
+|                                       | <api-utilities>`. By default, this is the major version of Kinto.        |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.version_prefix_redirect_enab  | By default, all endpoints exposed by Kinto are prefixed by a             |
+| led  ``True``                         | :ref:`version number <versioning>`. If this flag is enabled, the server  |
+|                                       | will redirect all requests not matching the supported version to the     |
+|                                       | supported one.                                                           |
++---------------------------------------+--------------------------------------------------------------------------+
+
+Example:
+
+.. code-block:: ini
+
+    cliquet.project_name = project
+    cliquet.project_docs = https://project.rtfd.org/
+    # cliquet.project_version = 1.0
+
+Application profiling
+=====================
+
+It is possible to profile the application while its running. This is especially
```

Comment:
I would replace this with a description of *what* the profiler will generate, and *how* that output can be used to debug the app.